dynamic-layers/recipes-kernel/linux/linux-variscite/0001-add-hmx-device-tree.patch

```diff
@@ -65,10 +65,12 @@ v16. Add wakeup-source property to tcan4x5x nodes. This is used by new
 v17. Rename gpio-leds identifiers for consistency and clarity.
      Update copyright years.
 
+v18. Enable PCI Express.
```

Comment:
fix and activate reference clock, also add pci_express reset pin (without it the modem will be in pci reset so not detected. somthing like that in the v18. also this is for 5G modem support (the pci clock was a warning in boot that has been removed by declare the refrence clock

Reply:
@RikardO-HM, thanks. Better now?